### ISSUE TITLE
[RCS1260] False Positive

### ISSUE THREAD

**Issue by viceroypenguin** [dotnet/roslynator#1439]
**Product and Version Used**: 
Roslynator.Analyzers

**Steps to Reproduce**:
set `.editorconfig` values:
```
roslynator_trailing_comma_style = omit_when_single_line
dotnet_diagnostic.RCS1260.severity = warning
```

**Actual Behavior**:
Warning RCS1260 on the `,` at the end of `Property1`
```csharp
var value = new Object
{
    Property1 = "Value"{|RCS1260:,|}
}
```

**Expected Behavior**:
No warning
```csharp
var value = new Object
{
    Property1 = "Value",
}
```

### Possible correction:

I believe the following code relies solely on the `initializer.Expressions` syntax node, when it should be based on the `initializer` code. 

https://github.com/dotnet/roslynator/blob/7505ae82a935905c8e4733f3c4bf719f6b5fb446/src/Analyzers/CSharp/Analysis/AddOrRemoveTrailingCommaAnalyzer.cs#L67

Other `IsSingleLine()` checks in this file need similar review.
